apps/frontend/src/app/shared/components/layout/navbar-pages/navbar-pages.component.html

```diff
@@ -252,47 +252,52 @@
               aria-labelledby="menu"
             >
               <div class="py-2">
-                <div
```

Comment:
I think the menu would look better if it was a little wider and it was aligned with the profile picture of the user, right now it is positioned too low and it feels and looks like it is floating, it should be connected to the profile picture. 

![image](https://github.com/Venture-Miner/wagademy/assets/111710515/24645498-2f87-4398-a4be-45eaba2c3652)